src/device-registry/utils/create-airqloud.js

```diff
@@ -272,49 +291,88 @@ const createAirqloud = {
   },
   refresh: async (request) => {
     try {
-      const { query } = request;
+      const { query, body } = request;
       const { tenant, id, name, admin_level } = query;
-      const responseFromFindSites = await createAirqloud.findSites(request);
-      if (responseFromFindSites.success === true) {
-        const sites = responseFromFindSites.data;
-        let requestForUpdateAirQloud = request;
-        requestForUpdateAirQloud["body"] = {};
-        requestForUpdateAirQloud["body"]["sites"] = sites;
-
-        const responseFromUpdateAirQloud = await createAirqloud.update(request);
-        if (responseFromUpdateAirQloud.success === true) {
-          return {
-            success: true,
-            message: responseFromUpdateAirQloud.message,
-            status: httpStatus.OK,
-            data: responseFromUpdateAirQloud.data,
-          };
-        }
-        if (responseFromUpdateAirQloud.success === false) {
-          const status = responseFromUpdateAirQloud.status
-            ? responseFromUpdateAirQloud.status
-            : "";
-          const errors = responseFromUpdateAirQloud.errors
-            ? responseFromUpdateAirQloud.errors
-            : "";
-          return {
-            success: false,
-            message: responseFromUpdateAirQloud.message,
-            status,
-            errors,
-          };
-        }
+
+      let requestForUpdateAirQloud = request;
+      requestForUpdateAirQloud["body"] = {};
```

Comment:
resolved runtime for AirQloud refresh. Just disabled the part of adding existing Sites and also add function to add center of an AirQloud. The part of adding Sites disabled due to unclear runtime errors